style.css

```diff
@@ -1,5 +1,5 @@
 @import "./view/font-jetbrains-mono";
 @import "./view/base";
 @import "./view/header";
-@import "./view/logo";
 @import "./view/page";
+@import "./view/preview"
```

Comment:
nit

Is this really supposed to be called `preview'? 😏

It seems that `preview` is more suitable for an image that briefly informs about the content.

Maybe we'll call it `logoAnimation`, `mainAnimation`, `startScreen`?

Reply:
I like preview because its not a complex word and it looks like something "pre")